cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2303,7 +2303,7 @@ macro(build_orc)
     endif()
     if("${COMPILER_VERSION}" VERSION_GREATER "4.0")
       set(ORC_CMAKE_CXX_FLAGS " -Wno-zero-as-null-pointer-constant \
--Wno-inconsistent-missing-destructor-override ")
```

Comment:
What breaks if this is warning is absent? Could you add a TODO so this can be removed when the issue is resolved in orc?

Reply:
I don't think we should be tracking upstream dependencies warning.